README.md

```diff
@@ -57,36 +80,75 @@ insert into db1.table1 values (1, 'a');
-- `kubectl port-forward svc/polaris-service -n polaris 8181:8181 8182:8182` - To create secure connections between a local machine and a pod within the cluster for both service and metrics endpoints.
-  - Currently supported metrics endpoints:
-    - localhost:8182/metrics
-    - localhost:8182/healthcheck
+
+#### Running in Docker
+
+Please note: there are no official Docker images for Apache Polaris yet. For now, you can build the
+Docker images locally.
+
+To build the Polaris server Docker image locally:
+
+```shell
+./gradlew clean :polaris-quarkus-server:assemble -Dquarkus.container-image.build=true
+```
+
+To run the Polaris server Docker image:
+
+```shell
+docker run -p 8181:8181 -p 8182:8182 apache/polaris:latest
+```
+
+#### Running in Kubernetes
+
+- `./run.sh` - To run Polaris as a mini-deployment locally. This will create a Kind cluster, 
+  then deploy one pod and one service. The service is available on ports `8181` and `8182`.
+- `kubectl port-forward svc/polaris-service -n polaris 8181:8181 8182:8182` - To create secure 
+  connections between a local machine and a pod within the cluster for both service and metrics 
+  endpoints.
+  - Currently supported metrics and health endpoints:
+    - http://localhost:8182/q/metrics
+    - http://localhost:8182/q/health
 - `kubectl get pods -n polaris` - To check the status of the pods.
 - `kubectl get deployment -n polaris` - To check the status of the deployment.
 - `kubectl describe deployment polaris-deployment -n polaris` - To troubleshoot if things aren't working as expected.
 
-Running regression tests
-- `./regtests/run.sh` - To run regression tests in another terminal.
-- `docker compose up --build --exit-code-from regtest` - To run regression tests in a Docker environment.
+#### Running regression tests
+
+Regression tests can be run in a local environment or in a Docker environment.
+
+To run regression tests locally, you need to have a Polaris server running locally, with the 
+`test` Authenticator enabled. You can do this by running Polaris as below:
+
+```shell
+java -Dquarkus.profile=test -jar quarkus/server/build/quarkus-app/quarkus-run.jar
+```
+
+Then, you can run the regression tests using the following command:
+
+```shell
+POLARIS_HOST=localhost ./regtests/run.sh
+```
+
+To run regression tests in a Docker environment, you can use the following command:
+
+```shell
+docker compose -f regtests/docker-compose.yml up --build --exit-code-from regtest
+```
+
+The above command will by default run Polaris with the Docker image `apache/polaris:latest`; if you
+want to use a different image, you can modify the `docker-compose.yaml` file prior to running it;
+alternatively, you can use the following commands to override the image:
+
+```shell
+cat <<EOF > /tmp/polaris-image.yml
+services: { polaris: { image: localhost:5001/apache/polaris:latest } }
+EOF
+docker compose -f regtests/docker-compose.yml -f /tmp/polaris-image.yml up --build --exit-code-from regtest
+```
```

Comment:
I'd prefer to not mention it in the main readme, either leaving for users/developers to figure out by themselves or moving this to the readme file under the directory `regtests`.